ci-operator/config/tnozicka/openshift-acme/master.json

```diff
@@ -0,0 +1,77 @@
+{
+  "base_images": {
+    "os": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "centos",
+      "tag": "7"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.10"
+  },
+
+  "canonical_go_repository": "github.com/tnozicka/openshift-acme",
+
+  "binary_build_commands": "make build GOFLAGS='-v'",
+  "test_binary_build_commands": "make build GOFLAGS='-v -race'",
+
+  "images": [
+    {
+      "from": "os",
+      "to": "openshift-acme",
+      "dockerfile_path": "images/Dockerfile",
+      "inputs": {
+        "bin": {
+          "as": ["bin"]
+        }
+      }
+    }
+  ],
+
+  "tests": [
+    {
+      "as": "verify",
+      "from": "bin",
+      "commands": "ARTIFACT_DIR=/tmp/artifacts make verify -k",
+      "artifact_dir": "/tmp/artifacts"
+    },
+    {
+      "as": "unit",
+      "from": "test-bin",
+      "commands": "ARTIFACT_DIR=/tmp/artifacts make test",
+      "artifact_dir": "/tmp/artifacts"
+    },
+    {
+      "as": "e2e",
+      "from": "bin",
+      "commands": "ARTIFACT_DIR=/tmp/artifacts make test-extended",
+      "artifact_dir": "/tmp/artifacts",
+      "inputs": {
+        "openshift-acme": {}
```

Comment:
This is very temporary. I need to somehow connect this dependency with the suite for spinning up cluster from the template; maybe it's somewhere in the docs, but doesn't seem obvious to me right now

Reply:
At the moment, e2e tests are defined at the Prow job level:

https://github.com/openshift/release/tree/master/ci-operator#end-to-end-tests